index.d.ts

```diff
@@ -0,0 +1,9 @@
+type ClassValue = string | number | ClassDictionary | ClassArray;
+
+interface ClassDictionary {
+	[id: string]: boolean;
+}
+
+interface ClassArray extends Array<ClassValue> { }
+
+export default function(...classes: ClassValue[]): string;
```

Comment:
most consumers are currently doing `import * as classNames from "classnames";`, and this will break that. actually I don't even think this is the shape of the export; see [this part of the DT readme](https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default) for guidance.

```sh
$ node -p 'require("classnames").default'
undefined
$ node -p 'require("classnames")'
[Function: classNames]
```